.github/workflows/deploy_preview_forks.yml

```diff
@@ -9,8 +9,8 @@ permissions:
 
 jobs:
   deploy-preview:
-    # target repository for pull_request is primer/react but source repository isn't primer/react
-    if: ${{ github.repository == 'primer/react' && github.event.pull_request.head.repo.full_name != 'primer/react' }}
+    # target repository for pull_request is different from source repository
+    if: ${{ github.repository != github.event.pull_request.head.repo.full_name }}
```

Comment:
does this work for forks of forks as well?

Reply:
It will certainly try.

Reply:
seems reasonable 👍 

looking at the diff, I think the issue could also be fixed by updating the first condition? Though that probably won't work on forks of forks.

```
${{ github.repository != 'primer/react'
```